### PR TITLE
(PC-12063) fix(GenericInfoPage): responsive layout

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12063

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| `VenueNotFound` | `QuitSignupModal` | `PhoneValidationTooManyAttemps` | `PhoneValidationTooManySMSSent` | `NotYetUnderageEligibility` |
| ----------------------: | ----------------------: | -----------------------: | ----------------------: | ----------------------: |
| image                                 | image| image |  image   |                               |

| `NotYetUnderageEligibility` | `VerifyEligiblity` | `UnderageAccountCreated` | `BookingConfirmation` | `BookingNotFound` |
| ----------------------: | ----------------------: | -----------------------: | ----------------------: | ----------------------: |
|                              |                              |  no change        |                              |                               |

| `ForceUpdate` | `NoCOntentError` | `QuitIdentitityCheckModal` | `NotEligibleEduConnect` | `BeneficiaryRequestSent` |
| ----------------------: | ----------------------: | -----------------------: | ----------------------: | ----------------------: |
| image|  not used nor needed |                                |                              |                               |

| `IdentityCheckEnd` | `IdentityCheckPending` | `IdentityCheckUnavailable` | `Maintenance` | `PageNotFound` |
| ----------------------: | ----------------------: | -----------------------: | ----------------------: | ----------------------: |
|                              |                               |                                |                              |                               |

| `OfferNotFound` | `ConfirmDeleteProfile` | `DeleteProfileSuccess` | `RecreditBirthdayNotification` | `LayoutExpiredLink` |
| ----------------------: | ----------------------: | -----------------------: | ----------------------: | ----------------------: |
|                              |                               |                                |                              |                               |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/ui/components/GenericInfoPage.tsx [hunk 1]
```diff
@@ -20,17 +19,18 @@ type Props = {
...
-    animationSize = getSpacing(45),
-    icon: Icon,
-    iconSize = getSpacing(25),
-    title,
-    flex = true,
-  } = props
-  const spacingMatrix: SpacingMatrix = Object.assign(defaultSpacingMatrix, props.spacingMatrix)
+export const GenericInfoPage: FunctionComponent<Props> = ({
+  children,
+  spacingMatrix,
```
Comment (antoinewg) [inline src/ui/components/GenericInfoPage.tsx:24]: `spacingMatrix`, `animation`, `animationSize` ne sont pas utilisés comme props.
`iconSize` à 1 seul endroit (`src/features/home/components/NoContentError.tsx`)

donc je pense qu'on peut aller plus loin et supprimer ces props

File: src/ui/components/GenericInfoPage.tsx [hunk 2]
```diff
@@ -41,38 +41,32 @@ export const GenericInfoPage: FunctionComponent<Props> = (props) => {
         </Helmet>
       )}
       <Background />
-      <ScrollView bounces={false} contentContainerStyle={scrollViewContentContainerStyle}>
-        <Spacer.Column numberOfSpaces={spacingMatrix.top} />
+      <Content>
```
Comment (antoinewg) [inline src/ui/components/GenericInfoPage.tsx:44]: personnellement j'aime bien avoir une ScrollView pour pouvoir interagir avec la page et si jamais ça dépasse on peut scroller mais c'est un avis perso.

**Review by antoinewg — APPROVED** [review]
top sinon

File: src/ui/components/GenericInfoPage.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/ui/components/GenericInfoPage.tsx:24]: `animation` est utilisé dans `UnderageAccountCreated` ainsi que `RecreditBirthdayNotification` donc je laisse celle ci

File: src/ui/components/GenericInfoPage.tsx (hunk 2 above)
Comment (kopax-polyconseil) [reply src/ui/components/GenericInfoPage.tsx:39]: C'est une demande de Manon, il faut voir cela avec elle ^^

**Pull request merged**